apps/passport-client/components/screens/AddScreen/JustAddScreen.tsx

```diff
@@ -26,16 +26,16 @@ export function JustAddScreen({
   const { error, pcd } = useDeserialized(request.pcd);
   const syncSettled = useIsSyncSettled();
 
-  const onAddClick = useCallback(async () => {
+  const onAddClick = useCallback(() => {
     try {
-      await dispatch({
+      dispatch({
         type: "add-pcds",
         pcds: [request.pcd],
         folder: request.folder
       });
       setAdded(true);
     } catch (e) {
-      await err(dispatch, "Error Adding PCD", e.message);
+      err(dispatch, "Error Adding PCD", e instanceof Error ? e.message : "");
```

Comment:
FYI we have the function `getErrorMessage` in the `util` package which basically does the thing you're doing with `e instanceof...` which i think could be nice to use here and elsewhere for succinctness